### PR TITLE
rename RESINRC_* to BALENARC_*

### PR DESCRIPTION
**Pull request by ab77**

### CODE REVIEW

File: env.list.example [hunk 1]
```diff
@@ -6,7 +6,7 @@ INITSYSTEM=off
 password=
 proxy_ssh_port=22
 requests_verify=False
-RESINRC_PROXY_URL=balena-devices.com
-RESINRC_RESIN_URL=balena-cloud.com
+BALENARC_PROXY_URL=balena-devices.com
+BALENARC_RESIN_URL=balena-cloud.com
```
Comment (lurch) [inline env.list.example:10]: Should that be `BALENARC_BALENA_URL` ?  :shrug:

Comment (ab77) [reply env.list.example]: Thank you.

File: qemu.robot [hunk 2]
```diff
@@ -16,8 +16,8 @@ Preparing test environment
   Set Suite Variable    ${application_name}    %{application_name}
   Should Not Be Empty   ${application_name}    msg=application_name variable cannot be blank
   Set Suite Variable    ${device_type}    %{device_type}
-  Set Suite Variable    ${RESINRC_RESIN_URL}    %{RESINRC_RESIN_URL}
-  Set Suite Variable    ${RESINRC_PROXY_URL}    %{RESINRC_PROXY_URL}
+  Set Suite Variable    ${BALENARC_BALENA_URL}    %{BALENARC_BALENA_URL}
+  Set Suite Variable    ${BALENARC_PROXY_URL}    %{BALENARC_PROXY_URL}
   Set Suite Variable    ${image}    %{image}
   File Should Exist     ${image}  msg="Provided images file does not exist"
   Set Suite Variable    ${application_repo}    https://github.com/resin-io/autohat-ondevice.git
```
Comment (lurch) [inline qemu.robot:23]: Does this URL also need updating? :thinking:  It currently redirects to https://github.com/balena-io/autohat-ondevice

Comment (ab77) [reply qemu.robot:23]: It's actually a critical piece of this stack of cards - don't ask me how I know :)

**Comment by ab77** [comment]
lgtm

**Comment by ab77** [comment]
lgtm

**Pull request merged**